package.json

```diff
@@ -11,7 +11,7 @@
     "cordova": "./bin/cordova"
   },
   "scripts": {
-    "test": "node node_modules/jasmine-node/bin/jasmine-node --captureExceptions --color spec",
+    "test": "node node_modules/jasmine/bin/jasmine --captureExceptions --color spec",
     "cover": "node node_modules/istanbul/lib/cli.js cover --root src --print detail node_modules/jasmine-node/bin/jasmine-node -- spec"
```

Comment:
@audreyso, looks like you'll also need to update `cover` command to use jasmine inatead of jasmine-node